### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by thepatrick**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/thepatrick/ReviewNinja-Welcome/pull/3" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -38,5 +38,3 @@ That’s it! These are the basic features of ReviewNinja to help you get
 started. We hope you enjoy using the tool, and please 
 [let us know](https://github.com/reviewninja/review.ninja/issues/new) 
 how we can make it better.
-
-This is a silly line
```
Comment (thepatrick) [inline ReadMe.md]: How dare you remove the sillness! !fix

Comment (thepatrick) [inline ReadMe.md]: Well ok then !fixed

Comment (thepatrick) [inline ReadMe.md]: No! !fix

Comment (thepatrick) [inline ReadMe.md]: Ok then !resolve

Comment (thepatrick) [inline ReadMe.md]: !fixed

**Pull request merged**